projects/core/src/lib/layers/legend/legend-raster/raster-legend.component.ts

```diff
@@ -56,6 +56,52 @@ export class CastMeasurementToContinuousPipe implements PipeTransform {
     }
 }
 
+export function unifyDecimals(values: number[]): number[] {
+    // Early return if all values differ by more than 1
+    if (values.every((x) => x >= 1)) {
+        const toCompare: number[] = values.map((x) => Math.floor(x)).sort((a, b) => a - b);
+        let oneApart = true;
+        for (let i = 0; i < toCompare.length - 1; i++) {
+            if (toCompare[i + 1] - toCompare[i] < 1) {
+                oneApart = false;
```

Comment:
if it is false, you can `break`

Reply:
why don't you make a helper function that returns a boolean and in that case returns early?

Reply:
Moved the code to a helper function and added an early return 330864ceb13c28994b236530b8cf6744a8428b7f